src/FileSystem/FilePathHelper.php

```diff
@@ -45,6 +45,15 @@ public function relativePath(string $fileRealPath): string
         return $this->relativeFilePathFromDirectory($fileRealPath, getcwd());
     }
 
+    public function absolutePath(string $fileRealPath): string
+    {
+        if ($this->filesystem->isAbsolutePath($fileRealPath)) {
+            return $fileRealPath;
+        }
+
+        return getcwd() . '/' . $fileRealPath;
```

Comment:
Just a quick look, `realpath()` should be used, directly use `getcwd()` may cause invalid path, for example, when using as standalone or global path

https://getrector.com/documentation/how-to-run-on-php-53

This must be tested carefully tho /cc @llaville